### PR TITLE
Depreciate com.kitware.F3D and rename com.kitware.F3D to com.f3d-app.F3D

### PR DESCRIPTION
**Pull request by kevinsmia1939**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. [**Link:**](https://github.com/f3d-app/f3d/issues/124#issuecomment-990358228)
- [] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
what is the rationale for renaming?

File: com.f3d-app.F3D.yaml [hunk 1]
```diff
@@ -0,0 +1,307 @@
+app-id: com.f3d-app.F3D
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '20.08'
```
Comment (hfiguiere) [inline com.f3d-app.F3D.yaml:4]:
```suggestion
runtime-version: '21.08'
```

File: com.f3d-app.F3D.yaml [hunk 2]
```diff
@@ -0,0 +1,307 @@
+app-id: com.f3d-app.F3D
```
Comment (hfiguiere) [inline com.f3d-app.F3D.yaml:1]:
```suggestion
app-id: com.f3d_app.F3D
```

It has to be with an underscore and not a `-`.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by kevinsmia1939** [comment]
> what is the rationale for renaming?
Dev suggest that ideally it should be rename.
https://github.com/f3d-app/f3d/issues/124#issuecomment-990358228

**Comment by hfiguiere** [comment]
as I mentionned there, `f3d-app.com` doesn't resolve. So it break requirement of being a "domain" under upstream control.

**Comment by hfiguiere** [comment]
The id should be `io.github.f3d_app.f3d`.

See https://github.com/f3d-app/f3d/issues/156

**Comment by kevinsmia1939** [comment]
i will close this one and

**Comment by hfiguiere** [comment]
you didn't need to close this.

**Pull request closed without merging**